neorusse/epic-mail
# [Chore #164375765] Setup Message model, controller and route

#### What does this PR do?
- Setup message models, controller, and route

#### Description of Task to be completed?
- A list of the major tasks you did to complete this feature/bug/chore
- Create message model using JS data structure
- Create controller for message model
- Create message route
- Enable app to use the message route
- Test route with postman

#### How should this be manually tested?
- Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
- Clone the repo
- Cd into the directory
- Run npm install
- Use postman to test routes

#### What are the relevant pivotal tracker stories?
- PT Story link - [#164375765](https://www.pivotaltracker.com/story/show/164375765)
